apis/apps/v1alpha1/cloneset_types.go

```diff
@@ -75,11 +75,30 @@ type CloneSetSpec struct {
 	Lifecycle *appspub.Lifecycle `json:"lifecycle,omitempty"`
 }
 
+// CloneSetDeletePriority defines the relationship among these conditions are ANDed
+type CloneSetDeletePriority struct {
+	// PodNames is the assemble of the pods’ names
+	PodNames []string `json:"podNames,omitempty"`
```

Comment:
podNames is duplicate with CloneSetScaleStrategy.PodsToDelete and not thus not necessary